wof/WaterML_1_1.py

```diff
@@ -402,7 +402,7 @@ def quote_xml(inStr):
         pos = mo.end()
     s3 = s1[pos:]
     s2 += quote_xml_aux(s3)
-    return s2
+    return s2.decode(ExternalEncoding)
```

Comment:
The logic I mentioned above it exactly to avoid that encode/decode whack-a-mole.

However, even though that would be the best practice, WOFpy was written a long time ago and doing this now would be an herculean job that is probably not worth it. Let's "beef up" the tests and fix things as they appear.